inline-r/src/Language/R/QQ.hs

```diff
@@ -134,7 +142,13 @@ collectAntis input = do
       -- Note: --slave was recently renamed to --no-echo. --slave still works
       -- but is no longer documented. Using the old option name for now just
       -- in case the user have an older (pre-2020) version of R.
-      "input_file <- \"" ++ input_file ++ "\"\n" ++
+      --                              
+      -- Change backslashes to forward slashes in tempFile names 
+      -- under Windows. Windows is tolerant of this Unixification, but 
+      -- Unix systems (and R) are less tolerant of naked backslashes 
+      -- outside of valid escape sequences. For example, 
+      -- str <- "C:\Users\joe" is invalid in R.
+      fixwinslash  $ "input_file <- \"" ++ input_file ++ "\"\n" ++
```

Comment:
Should `fixwinslash` be conditionally compiled?

Moreover, should it be only applied to `input_file` instead of the whole string?

Reply:
Yes, it is only required for Windows and should only be applied to input_file. The alternative is to get a temp file without backward slashes (or with doubled backward slashes).